### DISCUSSION TITLE
Greeting and other prompts

### DISCUSSION THREAD

**Discussion by shubhbhatia02** [FlowiseAI/Flowise#772] [General]
Hello all, I have a question about the welcome prompt. I want to embed the chatbot I created on a website but I also want to add welcome prompts and other prompts to input the user information. I'm unable to find information on how to do this.
What I want is when the user joins the chat, the bot greets the user and asks what it can help with. Then takes the user prompts provides information and lastly asks the user for their name and contact information. Would someone be able to tell me how I can achieve this? From what I understand I would need to connect my bot to a front end service (Kommunicate for example) where I can control the conversation flow, but would anyone else have any idea how to do this without using Kommunicate?

**Comment by HenryHengZJ** [comment]
for now you can't do that yet, but we have upcoming story to implement this. most people are using botpress voiceflow conversational chatbot builder to achieve this
